certified-connectors/IN-D Invoice Data Capture/apiDefinition.swagger.json

```diff
@@ -0,0 +1,473 @@
+                                      "description": "cgst"
+                                    },
+                                    "igst": {
+                                      "type": "object",
+                                      "properties": {
+                                        "value": {
+                                          "type": "string",
+                                          "description": "igst extracted from the invoice"
+                                        },
+                                        "conf_score": {
+                                          "type": "number",
+                                          "format": "float",
+                                          "description": "The confidence score for extracted igst from the invoice"
+                                        }
+                                      },
+                                      "description": "igst"
+                                    },
+                                    "vendor_gstin": {
+                                      "type": "object",
+                                      "properties": {
+                                        "value": {
+                                          "type": "string",
+                                          "description": "vendor gstin extracted from the invoice"
+                                        },
+                                        "conf_score": {
+                                          "type": "number",
+                                          "format": "float",
+                                          "description": "The confidence score for vendor gstin extracted from the invoice"
+                                        }
+                                      },
+                                      "description": "vendor_gstin"
+                                    },
+                                    "customer_gstin": {
+                                      "type": "object",
+                                      "properties": {
+                                        "value": {
+                                          "type": "string",
+                                          "description": "customer gstin extracted from the invoice"
+                                        },
+                                        "conf_score": {
+                                          "type": "integer",
+                                          "format": "int32",
+                                          "description": "The confidence score for customer gstin extracted from the invoice"
+                                        }
+                                      },
+                                      "description": "customer_gstin"
+                                    }
+                                  }
+                                },
+                                "description": "header_items"
```

Comment:
@gayathri35 please look through descriptions for responses, they should look like valid sentences. More information about responses can be found [here](https://docs.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance#response).